tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -4630,6 +4649,18 @@ SetpointManager:Scheduled,
   Always 20C, !- Schedule Name
   Water Cooling Coil Air Outlet Node;  !- Setpoint Node or NodeList Name
 
+        Schedule:Compact,
+          Always Point004,              !- Name
+          Any Number,              !- Schedule Type Limits Name
+          Through: 12/31,          !- Field 1
+          For: AllDays,            !- Field 2
+          Until: 24:00, 0.004;      !- Field 3
+
+        SetpointManager:Scheduled,
+          CW Coil Humidity Setpoint Manager, !- Name
+          MaximumHumidityRatio, !- Control Variable
+          Always 20C, !- Schedule Name
```

Comment:
Either this should be "Always Point004" or the new schedule isn't needed. The humrat SP gets written directly on the node so this must be needed for the new ManageSetPoints call?